RealmSwift/LinkingObjects.swift

```diff
@@ -274,7 +274,8 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
        is empty.
      */
     public func minimumValue<U: MinMaxType>(ofProperty property: String) -> U? {
-        return rlmResults.min(ofProperty: property) as! U?
+        guard let value = rlmResults.min(ofProperty: property) else { return nil }
```

Comment:
I'm not sure if I find that definitely clearer. Particularly, it's weird that in the optional cases we'll have `U.bridging` at the end but in the non-optional case (`sum`) we'll have
```swift
return U.bridging(rlmResults.sum(ofProperty: property)`
```